src/main/java/cubicchunks/server/ServerCubeCache.java

```diff
@@ -269,151 +263,156 @@ public boolean chunkExists(int cubeX, int cubeZ) {
 	//==============================
 
 	@Override
-	public boolean cubeExists(int cubeX, int cubeY, int cubeZ) {
-		long columnAddress = AddressTools.getAddress(cubeX, cubeZ);
-		Column column = this.loadedColumns.get(columnAddress);
-		if (column == null || column.unloaded) {
-			return false;
-		}
-		Cube cube = column.getCube(cubeY);
-		return cube != null && !cube.unloaded;
+	public Cube getCube(int cubeX, int cubeY, int cubeZ) {
+		return getCube(new CubeCoords(cubeX, cubeY, cubeZ));
```

Comment:
The hash map uses it for now :/
`private Map<CubeCoords, Cube> cubemap = new HashMap<>();`
Should I add a TODO for now? Or is there an int x,y,z to Object hash map? (I could make a custom one if need be)